scripts/apt/install_runtime_dependencies.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Install Darktable if building for a supported architecture
+if [ "${TARGETPLATFORM}" = "linux/amd64" ] || [ "${TARGETPLATFORM}" = "linux/arm64" ]; then \
+  apt-get update
+  apt-get install -y curl gnupg gpg
+
+  echo 'deb https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/ /' \
+    | tee /etc/apt/sources.list.d/graphics:darktable.list
+  curl -fsSL https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/Release.key \
+    | gpg --dearmor | tee /etc/apt/trusted.gpg.d/graphics_darktable.gpg > /dev/null
+
+  apt-get update
+  apt-get install -y darktable
+
+  apt-get purge -y curl gnupg gpg
+  apt-get autoremove -y
+fi
+
+apt-get update
+apt-get install -y libdlib19.1 ffmpeg exiftool libheif1 sqlite3
```

Comment:
I put `darktable` in the top because `darktable` is more special.

`purge` command only works with `darktable` session. The other part doesn't purge any packages. It's not part of cleanup commands, and it should be coupled with `darktable` session.

Reply:
Ok, I see your point and it makes sense.
However, I have 1 more thing to say) Each `apt-get update` takes time, and in your case you have to run it 3 times on a platform, supporting Darktable. If you put the generic tools installation on top, you could decrease the number of runs by 1 (as it was in Dockerfile)
A few sec faster)

But sure, it is not something critical at all

Reply:
Agreed on 3 times `apt update` wasted some time. But even in the previous `Dockerfile`, it's 2 times `apt update`, once before installing all packages, once before installing `darktable`. As `darktable` is installed from a 3rd repo, I can't find a way to run `apt update` only once and do them all together.

I updated and reduced to 2 times.

Reply:
Yes, as we support more platforms than Darktable, the `if` section for its installation (and the 2nd `apt update`) is unavoidable